Sources/tuist/main.swift

```diff
@@ -9,5 +9,4 @@ if CommandLine.arguments.contains("--verbose") {
 LogOutput.bootstrap()
 
 import TuistKit
-var registry = CommandRegistry()
-registry.run()
+TuistCommand.main()
```

Comment:
Does this mean that only the generate command works?

Reply:
Yes - it'd be too messy to keep merging the migration into `master`, that's why this PR points to `argument-parser` branch - I have tried to explain it more clearly [here](https://github.com/tuist/tuist/pull/1154#issue-394363041) in `Implementation Strategy`

Reply:
that also means `feature` tests will not pass, but we will make sure those work before merging into master